assets/queries/dockerfile/changing_default_shell_using_shell_command/metadata.json

```diff
@@ -1,9 +1,9 @@
 {
   "id": "8a301064-c291-4b20-adcb-403fe7fd95fd",
-  "queryName": "Changing Default Shell Using SHELL Command",
+  "queryName": "Changing Default Shell Using RUN Command",
   "severity": "MEDIUM",
   "category": "Insecure Defaults",
-  "descriptionText": "Using the command SHELL to override the default shell instead of the RUN command",
+  "descriptionText": "Using the command RUN to override the default shell instead of the SHELL command leads to ineficiencies. It also does not make sense since Docker provides the SHELL command for this exact purpose.",
```

Comment:
```suggestion
  "descriptionText": "Using the command RUN to override the default shell instead of the SHELL command leads to inefficiencies. It also does not make sense since Docker provides the SHELL command for this exact purpose.",
```

Reply:
This requested change is marked as resolved but the change was not done, @cxAndreFelicidade